ingenialink/canopen/network.py

```diff
@@ -246,7 +246,9 @@ def scan_slaves(self) -> List[int]:
             Containing all the detected node IDs.
 
         """
+        self._init_kvaser_lib()
```

Comment:
Can you do a scan while connected to another slave ?

1. Scan slaves
2. Connect to slave A
3. Scan slaves
4. Connect to Slave B
5. Disconnect from A and B

Reply:
Yes, you can, but if you, for example, add another transceiver, will not see changes. But this is a library problem, for this reason we must reload the lib in the scans if we can.